Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN \
         --only-binary=:all: \
         --find-links "${WHEELS_LINKS}" \
         -e ./homeassistant \
+        --config-settings editable_mode=compat \
```

Comment:
Why would we still do this at this point?

../Frenck

Reply:
Strictly speaking, it doesn't resolve any current issue. Although it would probably have helped with https://github.com/home-assistant/core/issues/95504.

Doing the editable install with a `.pth` file might also help with other things though as well, just because the "new" setuptools default for editable installs isn't well supported (by other tools).

Reply:
Right, but I'm missing the point on still doing it? As in #95504 has been resolved, and this is the Dockerfile for distribution, what would benefit from this at this point? What issue is this solving right now? Which tools would be problematic right now that are used in combination with the resulting image?

Reply:
I'm not aware of any current issues. This would just be a precautionary measure.

_If it's not worth, feel free to close it._

Reply:
In that case, I would say: Let's close it. If new issues/cases occur, we can still decide to re-open 👍 

../Frenck